spiceConcurrentCall_test.py

```diff
@@ -0,0 +1,109 @@
+# %%
+import spiceypy as spice # this imports the version 4.0.0
+import os
+
+# %%
+import functools
+import threading
+
+from spiceypy.spiceypy import spkezr
+
+_spicelock = threading.RLock()
+
+enablethreadinglock = False
+
+def spicelock_for_multithread(f):
```

Comment:
Test code should not re-implement the code to be tested.

Reply:
removed